src/lib/addClipMarkup.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Add clip-related markup to page head for Google SEO.
+ *
+ * @param {player} Player An instance of the Player.
+ * @return {void}
+ */
+// TODO: NEED TO DETERMINE WHERE THIS METHOD SHOULD BE DEFINED, IF NOT HERE
+export function addClipMarkup(player) {
+    if (!player) {
+        return;
+    }
+
+    // Any off-site page may have multiple embedded Vimeo videos. We can inject
+    // VideoObject markup for each video. According to the Google SEO team, we
+    // should include 'chapters' data for only one video in the page
+    const scriptElem = document.querySelector("script[type='application/ld+json']");
+    let existingMicrodataHasChapters;
+    let existingMicrodata = [];
+
+    if (scriptElem) {
+        const temp = JSON.parse(scriptElem.textContent);
```

Comment:
Make sure to account for parsing errors here.

Reply:
Updated 👍